applications/crossbar/doc/conference.md

```diff
@@ -247,6 +253,15 @@ curl -v -X PUT \
     http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/conferences/{CONFERENCE_ID}/participants/{PARTICIPANT_ID}
 ```
 
+When performing actions on participants, it's best practice to add a random string to the end of the call to ensure the request is viewed as 'unique'. For example:
```

Comment:
I don't understand this best practice? At best, you're cache-busting the browser but there's nothing advantageous in doing this from an API perspective. What's the thought here?

Reply:
Hey James,
Many times I'll get a HTTP/1.1 304 Not Modified for API calls to conferences for some reason in 4.0. I now add the random filter to make the request unique. Check out support ticket 29528